adaptive_n_back/experiment.js

```diff
@@ -200,6 +234,9 @@ for (var b = 0; b < num_blocks; b++) {
 		};
 		adaptive_n_back_experiment.push(test_block)
 	}
+	if ($.inArray(b,[4,7,15]) != -1) {
```

Comment:
b is an iterator over blocks. So this says "put in an attention-check block on blocks 5, 8 and 16"